src/screens/docs/components/victory-area/ecology.md

```diff
@@ -1,260 +1,410 @@
 # VictoryArea
 
-VictoryArea renders a dataset as a single area. VictoryArea can be composed with [VictoryChart] to create area charts.
+VictoryArea renders a dataset as a single area. VictoryArea can be composed with [`VictoryChart`] to create area charts.
 
 ```playground
-<VictoryArea
-  data={[
-    {month: "September", profit: 35000, loss: 2000},
-    {month: "October", profit: 44000, loss: 8000},
-    {month: "November", profit: 55000, loss: 5000}
-  ]}
-  x="month"
-  y={(datum) => datum.profit - datum.loss}
-/>
+<VictoryChart
+  theme={VictoryTheme.material}
+  style={{
```

Comment:
I see what you mean. I think I just wanted the first image on a given docs page to be instantly recognizable as "oh, that's what I want". I think wrapped and unwrapped is a good idea though.